app/components/layer-file/component.js

```diff
@@ -3,6 +3,12 @@ import Ember from 'ember';
 export default Ember.Component.extend({
     showSelect: false,
     noFileFound: true,
+    marginTop: Ember.computed('editMode', function(){
+        if(this.get('editMode')){
+            return 'margin-top';
+        }
+        return '';
+    }),
```

Comment:
Can this be applied as a class binding on the root element?

Reply:
@birdbrained could you give me an example